data_pipeline/meltano-plugins.yml

```diff
@@ -12,6 +12,9 @@ plugins:
     - name: tap-exchangeratehost
       variant: anelendata
       pip_url: git+https://github.com/anelendata/tap-exchangeratehost.git
+    - name: tap-jira
+      variant: meltanolabs
+      pip_url: git+https://github.com/jaceksan/tap-jira.git
```

Comment:
Consider adding commit sha or tag for the 1.0 version.

Reply:
Added a special branch.

I created PR to upstream repo. Once it's merged, I switch back to upstream.